cmd/manager/server.go

```diff
@@ -95,9 +91,9 @@ func run() {
 		log.Fatalf("Failed to get watch namespace: %v", err)
 	}
 
-	leaderElectionNS, err := getLeaderElectionNamespace()
-	if err != nil {
-		log.Fatalf("Failed to get leader election namespace: %v", err)
+	leaderElectionNS, leaderElectionEnabled := getLeaderElectionNamespace()
+	if !leaderElectionEnabled {
+		log.Warn("Leader election namespace not set. Leader election is disabled.")
```

Comment:
Would it be better to add more description to tell user that disabling leader election is not encouraged, and might be only used for debugging purpose.

Reply:
I've added a warning that this isn't appropriate for production use.